src/services/io/useIO.ts

```diff
@@ -741,7 +741,28 @@ export const useIO = create<IOStore>((set, get) => ({
 
   saveOpenTimelineIO: async () => {
 
-  }
+  },
+  
+  exportEntity: async (entity: ClapEntity) => {
+    const blob = new Blob([JSON.stringify(entity, null, 2)], { type: 'application/json' })
+    const fileName = `${entity.label.toLowerCase()}.clap`
+    get().saveAnyFile(blob, fileName)
+  },
+
+  importEntity: async (file: File): Promise<ClapEntity | undefined> => {
+    try {
+      const text = await file.text()
+      const entity = JSON.parse(text) as ClapEntity
+      if (!entity.id || !entity.label || !entity.category) {
+        throw new Error('Invalid entity file format')
+      }
+      return entity
+    } catch (error) {
+      console.error('Error importing entity:', error)
+      // You might want to show an error message to the user here
+      return undefined
+    }
+  },
```

Comment:
I'm going to change those, 

this code would work fine for a custom file format (eg. a file that would be called `some_actor.entity`)

but if we save to a `.clap`, then there is a specific structure to follow (store the entities inside the `entities` array) otherwise it would corrupt other apps

 I will make the change